installation/paas-deployments/d2cio/README.md

```diff
@@ -0,0 +1,25 @@
+# Deploying Rocket.Chat on D2C.io
```

Comment:
Please add this file to the [Table of Contents](https://github.com/RocketChat/docs/blob/master/_data/toc.yml)